config.nims

```diff
@@ -1,6 +1,3 @@
-when defined(macosx):
-  import algorithm, strutils
-
```

Comment:
This one needs double check. I assume there's a reason those are conditionally imported. But still, nim complains they aren't used... so maybe someone can verify this?